tf-dev-override.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+# required for go install terraform-provider-google-beta
+go mod download github.com/hashicorp/terraform-plugin-mux
+
+go install github.com/hashicorp/terraform-provider-google-beta
+
+# create terraform configuration file
+cat <<EOF > /tmp/tf-dev-override.tfrc
```

Comment:
What would you think about including this dev override file in the root directory and referencing it there instead of generating it?

Reply:
use sed to replace GOPATH?

Reply:
you're right - we can't hardcode a GOPATH. As an alternative maybe we could generate this file (in the current directory) only if it doesn't already exist? That would let users be able to modify it. What do you think?

Reply:
Done